src/site/stages/build/drupal/diff/README.md

```diff
@@ -0,0 +1,32 @@
+# GraphQL response vs. Tome data by Model
+
+[block_content](./block_content.md)
+
+[consumer](./consumer.md)
+
+[crop](./crop.md)
+
+[entity_subqueue](./entity_subqueue.md)
+
+[file](./file.md)
+
+[media](./media.md)
+
+[menu_link_content](./menu_link_content.md)
+
+[node](./node.md)
+
+[paragraph](./paragraph.md)
+
+[redirect](./redirect.md)
```

Comment:
Maybe we should have a different indicator to suggest that it _might_ be used, but we didn't explicitly find it in the graphql?